src/components/promoHeader.jsx

```diff
@@ -0,0 +1,53 @@
+/* eslint-env browser */
+import React, { Component } from 'react';
+
+
+class PromoHeader extends Component {
+  render() {
+    return (
+      <header className="promo-header bg--dark-purple">
+        <div className="promo-header__content">
+          <div className="promo-header__content-inner promo-header__content-inner--centre">
+            <h1>
+              <span className="visually-hidden">Giftaid it</span>
+            </h1>
+            <h2>
+              <span id="giftaid-form" className="bold">Fill in the short form </span>
+              <a
+                aria-labelledby="giftaid-form"
+                className="link link--light-grey inline"
+                href="#form"
+              >
+                below
+              </a>
+              <br />to Gift aid your donation and
+              <br className="mobile-line-break" />
```

Comment:
If you **_did_** want to remove this markup from the component itself, you can just import an HTML file like we said, check out this:

https://github.com/comicrelief/grants-eligibility-form/blob/master/src/Components/OutcomeMessage/OutcomeMessage.jsx

`import Parser from 'html-react-parser';`

`import m1 from './templates/m1.html';`

`OutcomeMessage.defaultProps = {
messages: [m1, m2, ...],
...`

`    return (
      <div className="outcome-message">
        {Parser(this.props.messages[currentMessage])}
...`